### ISSUE TITLE
Normal tag ({{ }}) doesn't add spaces for selected multiline text 

### ISSUE THREAD

**Issue by tylerdak** [austenc/vscode-blade-spacer#29]
Selecting text and adding {{ }} works great when the text is on a single line:
`example` => `{{ example }}`

However, doing the same with multiline text does not add the spaces around the text:
```
ex
am    
ple
```

yields

```
{{ex
am
ple}}
```

instead of

```
{{ ex
am
ple }}
```

**Comment by austenc** [comment]
Yeah, the multiline is tricky with the way the snippets (which drive the extension) work. I've tried to find a solution before but never landed on a good one yet. Sorry! Maybe we could do it in a separate command (that you could then add a shortcut key for). I'll think on it and see!
